src/sentry/snuba/sessions.py

```diff
@@ -176,7 +176,7 @@ def check_releases_have_health_data(
             Condition(Column("release"), Op.IN, release_versions),
         ],
     )
-    data = snuba.raw_snql_query(query, referrer="snuba.sessions.check_releases_have_health_data")[
+    data = snuba.raw_snql_query(query, referrer="snuba.sessions._check_releases_have_health_data")[
```

Comment:
this change should not be necessary

Reply:
yes, I oscillated on weather we should change the name in audits (since it is essentially the same function), I'll change it back.